### PR TITLE
Enabled and documented DEBUG option on Unix startup scripts

### PR DESCRIPTION
**Pull request by Mailaender**
Followup of https://github.com/OpenRA/OpenRA/pull/8282.

### CODE REVIEW

**Comment by pchote** [comment]
Wouldn't it be better to just include the `--debug` all the time?

**Comment by Mailaender** [comment]
It defaulting to true should be enough.

File: Makefile [hunk 1]
```diff
@@ -1,10 +1,10 @@
 ############################# INSTRUCTIONS #############################
 #
 # to compile, run:
-#   make
+#   make [DEBUG=true]
```
Comment (obrakmann) [inline Makefile]: People might assume that the default is false when you write it like that. I'd probably either invert that example usage, or add a line that states the default value.

Comment (Mailaender) [inline Makefile]: Inverted.

**Comment by obrakmann** [comment]
:+1:

**Comment by Mailaender** [comment]
Rebased after https://github.com/OpenRA/OpenRA/pull/8511 was merged prior to this.

**Comment by pchote** [comment]
Looks fine to me, although these changes don't apply to my platform

**Comment by pchote** [comment]
https://github.com/OpenRA/OpenRA/wiki/Changelog-%28bleed%29/_compare/d14896d84c8762d71afba17d5cd169233494dcf6%5E...d14896d84c8762d71afba17d5cd169233494dcf6

**Pull request merged**